LiteLoader/Kernel/Utils/NetworkHelper.cpp

```diff
@@ -22,7 +24,31 @@ void SplitHttpUrl(const std::string &url, string &host, string &path) {
     }
 }
 
-bool HttpGet(const string &url, const function<void(int, string)> &callback, int timeout) {
+httplib::Headers HandleHeaders(string headers)
+{
+    httplib::Headers maps;
+    Json::Reader reader;
+    Json::Value h;
```

Comment:
use nlohmann::json plz
非必要别用jsoncpp（）
```suggestion
    nlohmann::json j;
```

Reply:
骚瑞，我是第一次写 c++www